### ISSUE TITLE
Warn user or auto-split flags with space

### ISSUE THREAD

**Issue by coollog** [GoogleCloudPlatform/minikube-build-tools-for-java#19]
If the user uses a flag with a space in it, like:

```
minikubeStart {
  flags = ["--vm-driver none"]
}
```

The CommandExecutor will fail to run this command because the space is escaped (it will run `minikube start --vm-driver\ none`).

We can resolve this by warning the user that they have a space in the flag or by automatically splitting this into two parts for the command list.

**Comment by loosebazooka** [comment]
what if they want a flag parameter with spaces in it?
Are you tracking with `--`?

**Comment by coollog** [comment]
Currently, if they want a flag parameter with spaces in it, they would have to split up the parameter, like:

```
minikubeStart {
  flags = ["--vm-driver", "none"]
}
```

**Comment by loosebazooka** [comment]
oh right, but I mean `flags = ["--vm-driver", "my vm driver"]` <- auto escaping is a feature in this case

**Comment by coollog** [comment]
Yes, that should remain as a feature. Not sure what the best way to approach this may be. Maybe when executing a command, we can log it with the escape symbols, so like:

Instead of: `Running command : minikube --vm-driver none`
We have: `Running command : minikube --vm-driver\ none`

This is just a small ux improvement though.
